x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/blocklists.ts

```diff
@@ -110,13 +102,60 @@ export default function ({ getService }: FtrProviderContext) {
         },
       ];
 
+      const needsWritePrivilege: BlocklistApiCallsInterface = [
+        {
+          method: 'delete',
+          info: 'delete single item',
+          get path() {
+            return `${EXCEPTION_LIST_ITEM_URL}?item_id=${blocklistData.artifact.item_id}&namespace_type=${blocklistData.artifact.namespace_type}`;
+          },
+          getBody: () => undefined,
+        },
+      ];
+
+      const needsReadPrivilege: BlocklistApiCallsInterface = [
+        {
+          method: 'get',
+          info: 'single item',
+          get path() {
+            return `${EXCEPTION_LIST_ITEM_URL}?item_id=${blocklistData.artifact.item_id}&namespace_type=${blocklistData.artifact.namespace_type}`;
+          },
+          getBody: () => undefined,
+        },
+        {
+          method: 'get',
+          info: 'list summary',
+          get path() {
+            return `${EXCEPTION_LIST_URL}/summary?list_id=${blocklistData.artifact.list_id}&namespace_type=${blocklistData.artifact.namespace_type}`;
+          },
+          getBody: () => undefined,
+        },
+        {
+          method: 'get',
+          info: 'find items',
+          get path() {
+            return `${EXCEPTION_LIST_ITEM_URL}/_find?list_id=${blocklistData.artifact.list_id}&namespace_type=${blocklistData.artifact.namespace_type}&page=1&per_page=1&sort_field=name&sort_order=asc`;
+          },
+          getBody: () => undefined,
+        },
+        {
+          method: 'post',
+          info: 'list export',
+          get path() {
+            return `${EXCEPTION_LIST_URL}/_export?list_id=${blocklistData.artifact.list_id}&namespace_type=${blocklistData.artifact.namespace_type}&id=${blocklistData.artifact.id}`;
+          },
+          getBody: () => undefined,
+        },
+      ];
+
       describe('and has authorization to manage endpoint security', () => {
         for (const blocklistApiCall of blocklistApiCalls) {
           it(`should error on [${blocklistApiCall.method}] if invalid condition entry fields are used`, async () => {
             const body = blocklistApiCall.getBody();
 
             body.entries[0].field = 'some.invalid.field';
-            await supertest[blocklistApiCall.method](blocklistApiCall.path)
+            await supertestWithoutAuth[blocklistApiCall.method](blocklistApiCall.path)
+              .auth(ROLE.analyst_hunter, 'changeme')
```

Comment:
Consider updating this role to have a username and the default password so you don't have to duplicate the default password in every test. So it can be used as:
```
.auth(ROLE.analyst_hunter.username, ROLE.analyst_hunter.password)

Reply:
Interesting, we can work on this on a subsequent pr! Thanks for the suggestion!

Reply:
The fleet side does [something similar](https://github.com/ashokaditya/kibana/blob/6cc715d1e37f63a5712fc4fcd978c55c742298c6/x-pack/test/fleet_api_integration/apis/test_users.ts#L10) as I learned recently.